src/views/image-view.js

```diff
@@ -0,0 +1,10 @@
+const { View, BotTextMessage } = require('botfuel-dialog');
+
+class ImageView extends View {
```

Comment:
Maybe FileView will be more explicit. You trigger this one when you receive a file, not necessary a image